pkg/sentry/fs/g3doc/fuse.md

```diff
@@ -254,6 +254,52 @@ I/O syscalls like `read(2)`, `write(2)` and `mmap(2)`.
 -   `FUSE_BMAP`: Old address space API for block defrag. Probably not needed.
 -   `FUSE_NOTIFY_REPLY`: [TODO: what does this do?]
 
+## Benchmark FUSE
+
+FUSE benchmark makes FUSE syscall inside docker container to make sure required environment conditions
+are met - such as having the right libraries to start a FUSE server.
+
+### Setup
+
+To run benchmark:
+
+1. Make sure you have `Docker` installed
+2. Download all docker images `make load-all-images`
+3. Config `runsc` docker runtime to have VFS2 and FUSE supported. (e.g.  `make configure RUNTIME=runsc ARGS="--vfs2 --fuse ..." ...`)
+
+You should now have a runtime with the following options configured in
+`/etc/docker/daemon.json`
+```
+"runsc": {
+            "path": "path/to/your/runsc",
+            "runtimeArgs": [
+                "--debug-log",
```

Comment:
The argument right after `--debug-log` should be a path. It's OK but a little misleading here.